Dockerfile

```
@@ -52,7 +52,7 @@ COPY --from=buildenv /buildenv/lib/* /usr/local/lib64/
 COPY --from=buildenv /buildenv/src/games /usr/local/share/rcll-refbox/games
 COPY --from=buildenv /buildenv/src/msgs/*.proto /usr/local/share/rcll-refbox/msgs/
 COPY --from=buildenv /buildenv/src/libs/websocket/message_schemas/*.json /usr/local/share/rcll-refbox/libs/websocket/message_schemas/
-COPY --from=buildenv /buildenv/cfg/* /etc/rcll-refbox/
+COPY --from=buildenv /buildenv/cfg/. /etc/rcll-refbox/cfg/
```

That looks funny, is that necessary? Wouldn't something like the following work:
```suggestion
COPY --from=buildenv /buildenv/cfg /etc/rcll-refbox/
```

---

You are right, no need to create cfg subfolder within `/etc/`. Also the trailing `.` is not needed. I did use `*` initially as i was not familiar with Dockerfiles and i did not know that this flattens the first level of subdirectories :joy: 

Thanks for the feedback